src/app/modules/customer-management/components/customer-info/components/customer-list/customer-list.component.html

```diff
@@ -1,23 +1,23 @@
 <table
   *ngIf="customers"
-  class="table table-sm table-bordered table-striped mb-0"
+  class="table table-responsive-sm table-bordered table-striped mb-0"
   datatable
   [dtTrigger]="dtTrigger"
   [dtOptions]="dtOptions"
 >
   <thead class="thead-blue">
     <tr class="d-flex">
-      <th class="col-4">Customer ID</th>
-      <th class="col-5">Name</th>
-      <th class="col-3 text-center">Options</th>
+      <th class="col-sm-4">Customer ID</th>
```

Comment:
you can use only `col` to create responsive columns

Reply:
no, becouse the colums of customers page has a specific size